app/utility/list_datatable.rb

```diff
@@ -79,7 +79,8 @@ def list_entity_data(list_entity, interlock_ids = nil, list_interlock_ids = nil)
       industries: list_entity.entity.industries.join(','),    
       context: list_entity.list.custom_field_name.present? ? list_entity.custom_field : list_entity.entity.blurb,
       interlock_ids: interlock_ids,
-      list_interlock_ids: list_interlock_ids
+      list_interlock_ids: list_interlock_ids,
+      total_usd_donations: list_entity.entity.total_usd_donations
```

Comment:
This line adds a non-trivial amount of time to the SQL time since it runs an extra query for each entity on the list (we have lists with thousands of entities). 

Let's chat about future refactoring of  `ListDatatable`. I wrote recently a new rails backend for those tables and this might might be a good time to clean up this class.

Reply:
Yeah, I was thinking we would have to just deal with that with caching. Let's talk about this tomorrow.